client/src/App.tsx

```diff
@@ -29,6 +30,7 @@ function App(): JSX.Element {
                 <Route exact path="/signup" component={Signup} />
                 <AuthRoute exact path="/dashboard" component={Dashboard} />
                 <AuthRoute path="/profile/settings" component={Settings} />
+                <AuthRoute path="/booking" component={Booking} />
```

Comment:
I noticed the changes introduced for `AuthRoute` got lost somewhere along the way, I can't see the component under `main`. Let me know if this is the case and if you need help fixing it 👍🏼

Reply:
It should be marge from PR #49